src/MediaStreamTrack.ts

```diff
@@ -15,20 +16,23 @@ class MediaStreamTrack extends defineCustomEventTarget(...MEDIA_STREAM_TRACK_EVE
     _enabled: boolean;
     _settings: object;
     _muted: boolean;
+    _subscriptions: any[] = [];
+    _peerConnectionId: number;
 
-    id: string;
+    readonly id: string;
     kind: string;
     label: string;
     readyState: MediaStreamTrackState;
     remote: boolean;
-
+    
     constructor(info) {
         super();
 
         this._constraints = info.constraints || {};
         this._enabled = info.enabled;
         this._settings = info.settings || {};
         this._muted = false;
+        this._peerConnectionId = info.peerConnectionId;
```

Comment:
It is, and it is a valid use case, however I don't think it is currently supported in react-native-webrtc? For example, I know that track Ids are not necessarily unique for different peer connections, so if we want to be able to addTrack across different peer connections, we would need a unique identifier like a react tag or something similar.

Reply:
I think it's currently possible.

Loca Tracks are not referenced by PC but by their ID, which is guaranteed to be unique because we generate it (unlike the remote ones, for local ones The id and react tag are the same) using UUID v4.

Reply:
Thats a valid use case, but I am referring to an SFU use case. For example, adding a remote track from one peer connection to another. This might not be possible in this current setting.